src/style.css

```diff
@@ -1 +1,12 @@
-/* styles go here */
+.container {
+  margin-bottom: 20px;
+}
+
+.form-field {
+  margin-bottom: 10px;
+  border: none;
+}
+
+div:last-child {
```

Comment:
Don't style by tags. [Checklist](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md) the 6th point

Reply:
It's not fixed